src/GitHub.App/ViewModels/GitHubPane/PullRequestCreationViewModel.cs

```diff
@@ -82,8 +82,12 @@ public class PullRequestCreationViewModel : PanePageViewModelBase, IPullRequestC
                 .Where(_ => TargetBranch != null)
                 .Subscribe(x =>
                 {
+                    //// HACK: Why is `t` null?
+                    //if (!x.Any(t => t != null && t.Equals(TargetBranch)))
```

Comment:
Does this comment mean "Why was there a check for `t` being null here?"

Reply:
I temporarily added the `t != null` to allow the tests to pass. For some reason changing from using an interface to a class meant this come up null when testing.

I believe it was because NSubstitute automatically generates substitutes rather than returning `null` by default. I've since fixed this in the tests and removed the null check/comment.

Reply:
Could we remove the commented-out code here then, or is that fix in a separate PR?

Reply:
Oops, in my mind I'd removed the comments! ;-)